src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -243,9 +251,11 @@ public static void logAssertionStacktrace(AssertionError exception) {
       } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)